sheetfu/modules/table.py

```diff
@@ -112,8 +112,39 @@ def sort(self, field, reverse=False):
         if not self.items_range:
             return
         self.items.sort(key=lambda item: item.get_field_value(field), reverse=reverse)
-        for index, item in enumerate(self.items):
-            item.row_index = index
+        self._recalculate_item_indexes()
+        self._generate_full_items_range_batches()
+
+    def delete(self, instances_to_delete):
```

Comment:
Wouldn't it be clearer to have 2 methods: delete_items and delete.

delete would take list of indexes and delete them.
delete items would take a list of items, and pass their indexes to delete.

It's not necessary but I think it would be clearer and cleaner.

Is there a use case that would cause to send a list of mixed items and indexes?

Reply:
True, we lose support for passing mixed lists but I guess that's something that would never be useful xD